### ISSUE TITLE
QWenForCausalLM/transformer/vocab_embedding/embedding/GATHER_O_output_0: tensor volume exceeds 2147483647, dimensions are	[num tokens,8192]

### ISSUE THREAD

**Issue by zymy-chen** [NVIDIA/TensorRT-LLM#2204]
### System Info

GPU Name: 8 * H20
TensorRT-LLM : 0.11.0
NVIDIA-SMI 535.154.05 Driver Version: 535.154.05 CUDA Version: 12.4

### Information

- [x] The official example scripts
- [ ] My own modified scripts

### Tasks

- [ ] An officially supported task in the `examples` folder (such as GLUE/SQuAD, ...)
- [x] My own task or dataset (give details below)

### Reproduction

qwen2-72B,batch_size=30,input_len =8192 ,output_len=512
It works fine when running FP16, but FP8 comes up tensor volume exceeds 2147483647

### Expected behavior

![Image](https://github.com/user-attachments/assets/1d880111-bc25-4a0c-ae9c-93a584a9731a)

### actual behavior

trtllm-build --checkpoint_dir ./tllm_checkpoint_fp8--output_dir ./8-gpu/  --gemm_plugin float16 --max_batch_size 64 --max_input_len 256 --max_output_len 256 --remove_input_padding enable  --gpt_attention_plugin float16 --context_fmha  enable --workers 8 --tp_size 8  --paged_kv_cache enable  --use_paged_context_fmha enable --use_fused_mlp

### additional notes

None

**Comment by jershi425** [comment]
Hi @zymy-chen, this is a known issue for TRT version less than v10.1. Could you please check what is your TRT version?

**Comment by zymy-chen** [comment]
> 你好[@zymy-chen](https://github.com/zymy-chen)，这是 TRT 版本低于 v10.1 的已知问题。您能检查一下您的 TRT 版本吗？

TRT version is 10.2.0
![Image](https://github.com/user-attachments/assets/646f4668-08bd-4011-8f23-e5f005cbe68f)

**Comment by lfr-0531** [comment]
Can you try to rerun using the latest version of TensorRT-LLM to see if this issue persists?

**Comment by zymy-chen** [comment]
> Can you try to rerun using the latest version of TensorRT-LLM to see if this issue persists?

I try to TensorRT-LLM v0.12.0，but problem still exists

**Comment by zymy-chen** [comment]
Is there any plan to solve this problem?

**Comment by lfr-0531** [comment]
Still investigating. When you tried v0.12.0, did you update the docker image? Can you have a try again following the [installation doc](https://nvidia.github.io/TensorRT-LLM/installation/linux.html#installing-on-linux) to make sure the env is set up correctly?

**Comment by zymy-chen** [comment]
> Still investigating. When you tried v0.12.0, did you update the docker image? Can you have a try again following the [installation doc](https://nvidia.github.io/TensorRT-LLM/installation/linux.html#installing-on-linux) to make sure the env is set up correctly?

docker made using the dockerfile，
my docker env is:
cuda:
![Image](https://github.com/user-attachments/assets/ab0ffb6c-6f22-4282-8f85-13a8abb8c3ba)

python:
Package                   Version                   Editable project location
------------------------- ------------------------- ----------------------------------------------------
absl-py                   2.1.0
accelerate                0.34.2
aenum                     3.1.15
aiohttp                   3.9.5
aiosignal                 1.3.1
annotated-types           0.7.0
anyio                     4.6.0
apex                      0.1
argon2-cffi               23.1.0
argon2-cffi-bindings      21.2.0
asciitree                 0.3.3
asttokens                 2.4.1
astunparse                1.6.3
async-timeout             4.0.3
attrs                     23.2.0
audioread                 3.0.1
bandit                    1.7.7
beautifulsoup4            4.12.3
bleach                    6.1.0
blis                      0.7.11
build                     1.2.2
cachetools                5.3.3
catalogue                 2.0.10
certifi                   2024.7.4
cffi                      1.16.0
cfgv                      3.4.0
charset-normalizer        3.3.2
click                     8.1.7
click-option-group        0.5.6
cloudpathlib              0.18.1
cloudpickle               3.0.0
cmake                     3.30.0
colored                   2.2.4
coloredlogs               15.0.1
comm                      0.2.2
confection                0.1.5
contourpy                 1.2.1
coverage                  7.6.1
cuda-python               12.5.0
cudf                      24.4.0
cugraph                   24.4.0
cugraph-dgl               24.4.0
cugraph-equivariant       24.4.0
cugraph-pyg               24.4.0
cugraph-service-client    24.4.0
cugraph-service-server    24.4.0
cuml                      24.4.0
cupy-cuda12x              13.0.0
cutlass-library           3.5.1                    
cycler                    0.12.1
cymem                     2.0.8
Cython                    3.0.10
dask                      2024.1.1
dask-cuda                 24.4.0
dask-cudf                 24.4.0
dask-expr                 0.4.0
datasets                  2.19.2
debugpy                   1.8.2
decorator                 5.1.1
defusedxml                0.7.1
diffusers                 0.30.3
dill                      0.3.8
distlib                   0.3.8
distributed               2024.1.1
distro                    1.9.0
dm-tree                   0.1.8
einops                    0.8.0
entrypoints               0.4
evaluate                  0.4.3
exceptiongroup            1.2.1
execnet                   2.1.1
executing                 2.0.1
expecttest                0.1.3
fasteners                 0.19
fastjsonschema            2.20.0
fastrlock                 0.8.2
filelock                  3.15.4
flash-attn                2.4.2
fonttools                 4.53.1
frozenlist                1.4.1
fsspec                    2024.3.1
gast                      0.6.0
google-auth               2.32.0
google-auth-oauthlib      0.4.6
graphviz                  0.20.3
grpcio                    1.62.1
h11                       0.14.0
h5py                      3.10.0
httpcore                  1.0.5
httpx                     0.27.2
huggingface-hub           0.25.1
humanfriendly             10.0
hypothesis                5.35.1
identify                  2.6.1
idna                      3.7
igraph                    0.11.6
importlib_metadata        7.1.0
iniconfig                 2.0.0
intel-openmp              2021.4.0
ipykernel                 6.29.5
ipython                   8.21.0
ipython-genutils          0.2.0
janus                     1.0.0
jedi                      0.19.1
jieba                     0.42.1
Jinja2                    3.1.4
joblib                    1.4.2
json5                     0.9.25
jsonlines                 4.0.0
jsonschema                4.23.0
jsonschema-specifications 2023.12.1
jupyter_client            8.6.2
jupyter_core              5.7.2
jupyter-tensorboard       0.2.0
jupyterlab                2.3.2
jupyterlab_pygments       0.3.0
jupyterlab-server         1.2.0
jupytext                  1.16.2
kiwisolver                1.4.5
kvikio                    24.4.0
langcodes                 3.4.0
language_data             1.2.0
lark                      1.2.2
lazy_loader               0.4
librosa                   0.10.1
lightning-thunder         0.2.0.dev0
lightning-utilities       0.11.3.post0
lintrunner                0.12.5
llvmlite                  0.42.0
locket                    1.0.0
looseversion              1.3.0
marisa-trie               1.2.0
Markdown                  3.6
markdown-it-py            3.0.0
MarkupSafe                2.1.5
matplotlib                3.9.1
matplotlib-inline         0.1.7
mdit-py-plugins           0.4.1
mdurl                     0.1.2
mistune                   3.0.2
mkl                       2021.1.1
mkl-devel                 2021.1.1
mkl-include               2021.1.1
mock                      5.1.0
mpi4py                    3.1.5
mpmath                    1.3.0
msgpack                   1.0.8
multidict                 6.0.5
multiprocess              0.70.16
murmurhash                1.0.10
mypy                      1.11.2
mypy-extensions           1.0.0
nbclient                  0.10.0
nbconvert                 7.16.4
nbformat                  5.10.4
nest-asyncio              1.6.0
networkx                  3.3
ninja                     1.11.1.1
nltk                      3.9.1
nodeenv                   1.9.1
notebook                  6.4.10
numba                     0.59.1
numcodecs                 0.11.0
numpy                     1.24.4
nvfuser                   0.2.6a0+f73ff1b
nvidia-cudnn-frontend     1.5.1
nvidia-dali-cuda120       1.39.0
nvidia-modelopt           0.15.1
nvidia-nvimgcodec-cu12    0.2.0.7
nvidia-pyindex            1.0.9
nvtx                      0.2.5
nx-cugraph                24.4.0
oauthlib                  3.2.2
onnx                      1.16.0
openai                    1.39.0
opencv                    4.7.0
opt-einsum                3.3.0
optimum                   1.22.0
optree                    0.12.1
packaging                 24.0
pandas                    2.2.1
pandocfilters             1.5.1
parameterized             0.9.0
parso                     0.8.4
partd                     1.4.2
pbr                       6.1.0
pexpect                   4.9.0
pillow                    10.3.0
pip                       24.1.2
platformdirs              4.2.2
pluggy                    1.5.0
ply                       3.11
polygraphy                0.49.9
pooch                     1.8.2
pre-commit                3.8.0
preshed                   3.0.9
prometheus_client         0.20.0
prompt_toolkit            3.0.47
protobuf                  4.24.4
psutil                    5.9.8
ptyprocess                0.7.0
PuLP                      2.9.0
pure-eval                 0.2.2
py                        1.11.0
pyarrow                   14.0.2
pyarrow-hotfix            0.6
pyasn1                    0.6.0
pyasn1_modules            0.4.0
pybind11                  2.13.1
pybind11_global           2.13.1
pybind11-stubgen          2.5.1
pycocotools               2.0+nv0.8.0
pycparser                 2.22
pydantic                  2.8.2
pydantic_core             2.20.1
Pygments                  2.18.0
pylibcugraph              24.4.0
pylibcugraphops           24.4.0
pylibraft                 24.4.0
pylibwholegraph           24.4.0
pynvjitlink               0.2.3
pynvml                    11.5.3
pyparsing                 3.1.2
pyproject_hooks           1.1.0
pytest                    8.1.1
pytest-cov                5.0.0
pytest-flakefinder        1.1.0
pytest-forked             1.6.0
pytest-rerunfailures      14.0
pytest-shard              0.1.2
pytest-xdist              3.6.1
python-dateutil           2.9.0.post0
python-hostlist           1.23.0
pytorch-triton            3.0.0+989adb9a2
pytz                      2024.1
PyYAML                    6.0.1
pyzmq                     26.0.3
raft-dask                 24.4.0
rapids-dask-dependency    24.4.0a0
referencing               0.35.1
regex                     2024.5.15
requests                  2.32.3
requests-oauthlib         2.0.0
rich                      13.7.1
rmm                       24.4.0
rouge                     1.0.1
rouge-score               0.1.2
rpds-py                   0.19.0
rsa                       4.9
safetensors               0.4.5
scikit-learn              1.5.1
scipy                     1.13.1
Send2Trash                1.8.3
sentencepiece             0.2.0
setuptools                68.2.2
shellingham               1.5.4
six                       1.16.0
smart-open                7.0.4
sniffio                   1.3.1
sortedcontainers          2.4.0
soundfile                 0.12.1
soupsieve                 2.5
soxr                      0.3.7
spacy                     3.7.5
spacy-legacy              3.0.12
spacy-loggers             1.0.5
srsly                     2.4.8
stack-data                0.6.3
stevedore                 5.3.0
StrEnum                   0.4.15
sympy                     1.13.0
tabulate                  0.9.0
tbb                       2021.13.0
tblib                     3.0.0
tensorboard               2.9.0
tensorboard-data-server   0.6.1
tensorboard-plugin-wit    1.8.1
tensorrt                  10.3.0
tensorrt-llm              0.12.0
terminado                 0.18.1
texttable                 1.7.0
thinc                     8.2.5
threadpoolctl             3.5.0
thriftpy2                 0.5.0
tinycss2                  1.3.0
tokenizers                0.19.1
tomli                     2.0.1
toolz                     0.12.1
torch                     2.4.0a0+3bcc3cddb5.nv24.7
torch-tensorrt            2.5.0a0
torchvision               0.19.0a0
tornado                   6.4
tqdm                      4.66.4
traitlets                 5.9.0
transformer-engine        1.8.0+37280ec
transformers              4.42.4
treelite                  4.1.2
typer                     0.12.3
types-dataclasses         0.6.6
typing_extensions         4.8.0
tzdata                    2024.1
ucx-py                    0.37.0
urllib3                   2.0.7
virtualenv                20.26.5
wasabi                    1.1.3
wcwidth                   0.2.13
weasel                    0.4.1
webencodings              0.5.1
Werkzeug                  3.0.3
wheel                     0.43.0
wrapt                     1.16.0
xdoctest                  1.0.2
xgboost                   2.0.3
xxhash                    3.5.0
yarl                      1.9.4
zarr                      2.18.2
zict                      3.0.0
zipp                      3.19.0

**Comment by karljang** [comment]
Closing the issue as stale. If the problem persists, please feel free to open a new one.
And, please note that, since the 1.0 release, TensorRT-LLM has transitioned to the PyTorch workflow as the recommended approach, which no longer relies on trtllm-build.
